src/compiler/CMakeLists.txt

```diff
@@ -80,7 +80,8 @@ if(ROCM_FOUND)
                 ${ROCM_LIBRARIES}
                 -L${ROCM_LIBRARIES_DIR}/../hsa/lib -L${ROCM_LIBRARIES_DIR}
                 -Wl,--enable-new-dtags
-                -Wl,-rpath,\$ORIGIN/../hsa/lib -Wl,-rpath,\$ORIGIN/../lib
+                -Wl,-rpath,\$ORIGIN/../../hsa/lib -Wl,-rpath,\$ORIGIN/../../lib
+                -Wl,-rpath,${ROCM_LIBRARIES_DIR}/../hsa/lib -Wl,-rpath,${ROCM_LIBRARIES_DIR}
```

Comment:
Why do we need multiple run paths for library dependencies if all of the libraries are colocated in the same dir? How above just:
`-Wl,-rpath,\$ORIGIN`
and remove the rest?
Does anything in AOMP break with just $ORIGIN?
The issue is that ATMI package is also build and shipped as standalone atmp*rpm/deb that gets installed under /opt/rocm/atmi/lib, in which case, it needs the relative paths to other ROCm installed libraries. But since aomp uses its own version/copy of the libraries installed under aomp/lib, just $ORIGIN should suffice. Or atleast, add $ORIGIN first to the -rpath, so it works for both cases.